packages/core/src/returns/client/getPickupCapabilities.js

```diff
@@ -1,37 +1,24 @@
 import client, { adaptError } from '../../helpers/client';
 import join from 'proper-url-join';
 
-/**
- * @typedef {object} GetPickupCapabilitiesQuery
- *
- * @alias GetPickupCapabilitiesQuery
- * @memberof module:returns/client
- *
- * @property {Date} [pickupDay] - Timestamp for the day of pickup.
- * @property {string} [guestOrderId] - Order identifier. Only required if
- * the user is not registered (guest).
- * @property {string} [guestUserEmail] - User email. Only required if
- * the user is not registered (guest).
- */
-
 /**
  * Obtains the pickup capabilities for a specific order.
  *
  * @function getPickupCapabilities
  * @memberof module:returns/client
  *
- * @param {string} id - Return identifier.
- * @param {GetPickupCapabilitiesQuery} query - Query parameters.
+ * @param {number} id - Return identifier.
+ * @param {string} pickupDay - Day of the pickup. Format: YYYY-MM-DD.
  * @param {object} [config] - Custom configurations to send to the client
  * instance (axios).
  *
  * @returns {Promise} Promise that will resolve when the call to
  * the endpoint finishes.
  */
-export default (id, query, config) =>
+export default (id, pickupDay, config) =>
```

Comment:
This is breaking change, you should validate if it receives an query object, if it does you should use the legacy endpoint if it doesn't it should point to the new endpoint (`/account/v1/returns/`)

Reply:
done, now it supports both legacy and new endpoint.